ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -166,6 +166,12 @@ export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) => (
         <Allele isAlt={genotype.numAlt > (isHemiX ? 0 : 1)} variant={variant} textAlign="right" />
         /
         {isHemiX ? '-' : <Allele isAlt={genotype.numAlt > 0} variant={variant} textAlign="left" />}
+        {genotype.mitoCn && (
+          <span>
```

Comment:
can we change the `copyNumberGenotype` helper to include the `br` so there is less copy-paste happening?